skymp5-functions-lib/src/types/mp.d.ts

```diff
@@ -113,7 +113,7 @@ export interface Mp {
   get(formId: 0, propertyName: 'onlinePlayers'): number[];
   get(formId: number, propertyName: 'isDead'): boolean;
   get(formId: number, propertyName: 'worldOrCellDesc'): string;
-  get(formId: number, propertyName: 'percentages'): { health: number, magicka: number, stamina: number };
+  get(formId: number, propertyName: 'percentages'): { health: number; magicka: number; stamina: number };
```

Comment:
Was this change necessary?

Reply:
Oops, I have automatic formatting in VS Code with prettier